src/Mailhog_Command.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/**
+ * Enables/Disables admin-tools on a site.
+ *
+ * ## EXAMPLES
+ *
+ *     # Enable admin tools on site
+ *     $ ee admin-tools up example.com
+ *
+ * @package ee-cli
+ */
+
+use EE\Model\Site;
+use \Symfony\Component\Filesystem\Filesystem;
+
+class Mailhog_Command extends EE_Command {
+
+	/**
+	 * @var array $db Object containing essential site related information.
+	 */
+	private $db;
```

Comment:
Please change this variable name if it's used for site info and not the database. Maybe $site_data would be more suitable or $site. 
$db is look like it's for getting db instance.